internal/configconverter/ballast_remover.go

```diff
@@ -0,0 +1,45 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configconverter
+
+import (
+	"log"
+	"regexp"
+
+	"go.opentelemetry.io/collector/config/configparser"
+)
+
+var ballastKeyRegexp *regexp.Regexp
+
+func init() {
```

Comment:
If `RemoveBallastKey` is only called once is there a benefit to doing this in an init if there's only one consumer of `ballastKeyRegexp`?

Reply:
Yeah, there's no benefit. I'll move it to the function. (I hate `init()`s anyway 😛)